src/org/isf/examination/service/ExaminationIoOperationRepository.java

```diff
@@ -3,15 +3,17 @@
 import java.util.List;
 
 import org.isf.examination.model.PatientExamination;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.data.jpa.repository.Query;
 import org.springframework.data.repository.query.Param;
 import org.springframework.stereotype.Repository;
 
 @Repository
 public interface ExaminationIoOperationRepository extends JpaRepository<PatientExamination, Integer> {
-    @Query(value = "SELECT * FROM PATIENTEXAMINATION WHERE PEX_PAT_ID = :id ORDER BY PEX_DATE DESC", nativeQuery= true)
-    List<PatientExamination> findAllByIdOrderDesc(@Param("id") int id);
-    @Query(value = "SELECT * FROM PATIENTEXAMINATION WHERE PEX_PAT_ID = :id ORDER BY PEX_DATE DESC LIMIT :limit", nativeQuery= true)
-    List<PatientExamination> findAllByIdOrderDescLimited(@Param("id") int id, @Param("limit") int limit);
+  	@Query(value = "select p from PatientExamination p where p.patient.code = :patientCode")
+		List<PatientExamination> findByPatient_CodeOrderByPexDateDesc(@Param("patientCode") int patientCode);
+		@Query(value = "select p from PatientExamination p where p.patient.code = :patientCode")
+    Page<PatientExamination> findByPatient_CodeOrderByPexDateDesc(@Param("patientCode") int patientCode, Pageable pageable);
```

Comment:
it seems like some indentation issues for these lines?

also, are not javadoc comments required for OP-core JpaRepository interfaces? or JPA syntax is enough self-descriptive?

Reply:
With #50 the author "added a .editorconfig file, so all editorconfig-aware editors will use tabs from now on."

@emecas Hi think the second, JPA syntax is purposely self-descriptive